### PR TITLE
#184581317 Configuring connection to postgres  database 

### PR DESCRIPTION
**Pull request by Patienceineza**
#### What does this PR do?

this PR is for Configuring  The App to use PostgreSQL as DB Engine and Sequelize ORM]

####  Description of Task to be completed?

this task is associated with creating a folder and setting your connection to that database in that folder setting models and migration and all other important aspects to the database and its tables 

#### How should this be manually tested?

. this can be tested by 

- cloning this repo
- type `npm install `in your terminal

- adding the DATABASE_URL in your .env file in this format

-  postgres://username:password@127.0.0.1:port/database_name

#### Any background context you want to provide?

N/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184581317

#### Screenshots (if appropriate)

N/a

### CODE REVIEW

**Review by bencyubahiro77 — APPROVED** [review]
use PR template

**Review by INValentin — APPROVED** [review]
Checked it and it worked!

**Review by blackd44 — APPROVED** [review]
I see hound is bringing error `process not defined` but from me there is no problem,
it LGTM

File: src/database/config/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import dotenv from 'dotenv'
+dotenv.config()
+
+export default{
+  development: {
+    url: process.env.DEV_DATABASE_URL,
```
[the remark this answers is not in the document]
Comment (blackd44) [reply src/database/config/config.js:6]: i see no problem with this

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Please solve conflicts

**Comment by Patienceineza** [comment]
just resolved the conflicts please review

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Pull request closed without merging**